### ISSUE TITLE
Page Layout Mappings error 

### ISSUE THREAD

**Issue by oallain** [netgen-layouts/layouts-sylius#3]
Sylius version : 1.9
`netgen/layouts-sylius` version :  dev-master

In BO page **Layout mappings** stay blanc (no content loaded). In console i got error :
```
Uncaught TypeError: Cannot read property 'style' of undefined
    at e.value (app.js?472b384:formatted:22409)
    at e.value (app.js?472b384:formatted:22181)
    at e.value (app.js?472b384:formatted:22165)
    at new e (app.js?472b384:formatted:22157)
    at app.js?472b384:formatted:22561
    at app.js?472b384:formatted:22571
```
**More detail** :
![image](https://user-images.githubusercontent.com/6593252/115924396-f1d4f780-a47f-11eb-9ff4-7d1daa897c26.png)

**Step to reproduce :**
Install Sylius, `netgen/layouts-standard` and `netgen/layouts-sylius`.
Load Sylius fixtures and login to BO.
Then go to Layout mappings page.

**Comment by emodric** [comment]
Hi!

`dev-master` versions of Netgen Layouts cannot be used directly since they do not contain any built assets. You would have to build them yourself, across every project included in Netgen Layouts which is not a trivial task. Only tagged versions of the packages contain the built assets and are ready for use.

If you're using `netgen/layouts-sylius` `dev-master` due to Sylius 1.9 support, please be patient a little bit more, final version of Layouts 1.3 with Sylius 1.9 is just around the corner :)

**Comment by oallain** [comment]
Hello,

Yes, I want to use it with Sylius 1.9 🙂

Thanks for answering and for your good job on this plugin 💪

**Comment by emodric** [comment]
@oallain You might wanna know that integration of Layouts and Sylius is now released in version 1.3, so you can try it out with Sylius 1.9 :)

I will close this issue, and if any other issues arise, do not hesitate to open new issues!
